### PR TITLE
set max extent sample

### PR DESCRIPTION
**Pull request by prupani-7**
## Description

PR to add a new Kotlin sample "Set max extent" in Maps and Scenes category.
Using this sample, you can limit the view of a map to a particular area.

## Links and Data

Sample Epic: `runtime/kotlin/issues/1698` 

## What To Review

-  Review the code to make sure it is easy to follow like other samples on Android
- `README.md` and `README.metadata.json` files

## How to Test

Run the sample on the sample viewer or the repo.

### CODE REVIEW

File: set-max-extent/src/main/java/com/esri/arcgismaps/sample/setmaxextent/MainActivity.kt [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    private val extentSwitch: SwitchCompat by lazy {
+        activityMainBinding.extentSwitch
+    }
+
+    private val extentTextView: TextView by lazy {
+        activityMainBinding.extentText
+    }
+
+    private val extentEnvelope =
+        Envelope(Point(-12139393.2109, 5012444.0468), Point(-11359277.5124, 4438148.7816))
```
Comment (shubham7109) [inline set-max-extent/src/main/java/com/esri/arcgismaps/sample/setmaxextent/MainActivity.kt:58]:
```suggestion
    private val extentEnvelope = Envelope(
        Point(-12139393.2109, 5012444.0468),
        Point(-11359277.5124, 4438148.7816)
    )
```
Just personal preference :)

**Review by shubham7109 — APPROVED** [review]
Looks good! @prupani-7  👍

File: set-max-extent/README.metadata.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    "ignore": false,
+    "images": [
+        "set-max-extent.png"
+    ],
+    "keywords": [
+        "extent",
+        "limit panning",
+        "map",
+        "mapview",
+        "max extent",
```
Comment (hud10837) [inline set-max-extent/README.metadata.json:14]: I think "max" on its own would be a good keyword, but would that be included with this one?

Comment (shubham7109) [reply set-max-extent/README.metadata.json:14]: Yes, it should!

**Comment by prupani-7** [comment]
Thanks, for the review @hud10837 and @shubham7109. Merging in v.next

**Pull request merged**